crates/bitwarden-core/src/auth/login/access_token.rs

```diff
@@ -127,7 +127,7 @@ fn load_tokens_from_state(
         if time_till_expiration > 0 {
             let organization_id: Uuid = organization_id
                 .parse()
-                .map_err(|_| "Bad organization id.")?;
+                .map_err(|_| LoginError::BadOrganizationId)?;
```

Comment:
Can we rename this to something else while we're at it? It's not a bad id as much as it's invalid.